phoenicis-containers/src/main/java/org/phoenicis/containers/GenericContainersManager.java

```diff
@@ -170,33 +170,35 @@ public void deleteContainer(ContainerDTO container, Consumer<ContainerDTO> onSuc
 
         if (containerDirectories != null) {
             for (File containerDirectory : containerDirectories) {
-                final ConfigFile configFile = compatibleConfigFileFormatFactory
-                        .open(new File(containerDirectory, "phoenicis.cfg"));
-                final String containerPath = containerDirectory.getAbsolutePath();
-                final String containerName = containerPath.substring(containerPath.lastIndexOf('/') + 1);
-
-                // find shortcuts which use this container
-                List<ShortcutDTO> shortcutDTOS = libraryManager.fetchShortcuts().stream()
-                        .flatMap(shortcutCategory -> shortcutCategory.getShortcuts().stream()).filter(shortcut -> {
-                            boolean toAdd = false;
-                            try {
-                                final Map<String, Object> shortcutProperties = objectMapper
-                                        .readValue(shortcut.getScript(), new TypeReference<Map<String, Object>>() {
-                                        });
-                                toAdd = shortcutProperties.get("winePrefix").equals(containerName);
-                            } catch (IOException e) {
-                                LOGGER.warn("Could not parse shortcut script JSON", e);
-                            }
-                            return toAdd;
-                        }).collect(Collectors.toList());
-
-                if (directory.getName().equals("wineprefix")) {
-                    containers.add(new WinePrefixContainerDTO.Builder().withName(containerDirectory.getName())
-                            .withPath(containerPath).withInstalledShortcuts(shortcutDTOS)
-                            .withArchitecture(configFile.readValue("wineArchitecture", ""))
-                            .withDistribution(configFile.readValue("wineDistribution", ""))
-                            .withVersion(configFile.readValue("wineVersion", ""))
-                            .build());
+                if (!containerDirectory.isHidden()) {
```

Comment:
```Java
if (!containerDirectory.isHidden()) {
```
is the only change. The rest is indentation.

Reply:
By indentation you mean spaces from/to tabs?

Reply:
No. The code is in the `if` now, so it's indented one level more.